src/modules/commander/commander_params.c

```diff
@@ -243,9 +243,11 @@ PARAM_DEFINE_FLOAT(COM_DISARM_LAND, 2.0f);
 PARAM_DEFINE_FLOAT(COM_DISARM_PRFLT, 10.0f);
 
 /**
- * GPS preflight check
+ * Arming without GNSS configuration
  *
- * Measures taken when a check defined by EKF2_GPS_CHECK is failing.
+ * Configures whether arming is allowed without GNSS, for modes that require a global position
+ * (specifically, in those modes when a check defined by EKF2_GPS_CHECK fails).
```

Comment:
This makes it clear that the modes affected are those that rely on global position. The ones that rely on local position seem to not matter https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp#L279